pkg/config/autodiscover/autodiscover_targets.go

```diff
@@ -323,6 +326,37 @@ func getConfiguredOperatorTests() []string {
 	return opTests
 }
 
+// getBundleAndIndexImage provides the bundle image and index image for a given CSV.
+// These variables are saved in the `configsections.Operator` in order to be used by DCI,
+// which obtains them from the claim.json and provides them to preflight suite.
+func getBundleAndIndexImage(csvName, csvNamespace string) (bundleImage, indexImage string) {
+	// First step is to extract the installplan related to the csv
+	installPlanCmd := fmt.Sprintf("oc get installplan -n %s | grep %q | awk '{ print $1 }'", csvNamespace, csvName)
+	installPlan := execCommandOutput(installPlanCmd)
+
+	// If no installPlan is obtained, just return an empty string for the two returned variables
+	if installPlan == "" {
+		bundleImage = ""
+		indexImage = ""
+	} else {
```

Comment:
Fast return here, creating a custom error:
```
	if installPlan == "" {
        return "", "",  errors.New("installplan not found")
    }
```
After this fast return, you don't need the "else" branch, which also saves one indentation level.

Reply:
done